tests/components/zwave_js/test_init.py

```diff
@@ -1154,3 +1154,133 @@ async def test_node_model_change(hass, zp3111, client, integration):
     state = hass.states.get(custom_entity)
     assert state
     assert state.name == "Custom Entity Name"
+
+
+async def test_disabled_entity_on_value_removed(hass, zp3111, client, integration):
+    """Test that when entity primary values are removed the entity is disabled."""
```

Comment:
Let's call it removed instead of disabled. We remove the entities. Disabling an entity is something else.

Reply:
Thanks, was mixing up disabled and unavailable.